src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +174,21 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_output_element_type(const Napi::CallbackInfo& info) {
+    static const std::vector<os::js::Signature> allowed_signatures = {{1, {os::js::ValueType::Number}}};
+
+    if (ov::js::validate<Napi::Number>(info, allowed_signatures)) {
```

Comment:
Please move it inside try{}catch{} block to be consistent with other parts of code